apps/studio/components/grid/components/formatter/ForeignKeyFormatter.tsx

```diff
@@ -56,18 +57,17 @@ export const ForeignKeyFormatter = (props: Props) => {
         <Tooltip.Root delayDuration={0}>
           <Tooltip.Trigger asChild>
             <Button
+              asChild
               type="default"
               size="tiny"
               className="translate-y-[2px]"
-              onClick={() => {}}
               style={{ padding: '3px' }}
-              asChild
             >
               <Link
                 href={`/project/${projectRef}/editor/${targetTable?.id}?filter=${relationship?.target_column_name}%3Aeq%3A${value}`}
                 onClick={() => snap.setSelectedSchemaName(relationship.target_table_schema)}
               >
-                <IconArrowRight size="tiny" />
```

Comment:
don't need size

Reply:
when an icon is in a `<Button />` that is

Reply:
i think in this case we need the size? since the icon is used standalone

with size:
![image](https://github.com/user-attachments/assets/c09cafed-5ab0-4acb-8649-d58cda03cbcd)

without size:
![image](https://github.com/user-attachments/assets/93f12c3d-b1ca-4e02-8858-f18fc276d484)